common/deposits/hfe_deposit_features.txt

```diff
@@ -15,6 +15,91 @@ hfe_deposit_features_spiral_of_death_0 = {
 			consumer_goods = 5
 		}
 	}
+	drop_weight = {
+		weight = 0
+	}
+}
+
+HFE_TSP_exodus_start_point = {
+	station = shipclass_mining_station
+	is_for_colonizeable = no
+	resources = {
+		category = orbital_mining_deposits
+		produces = {
+			society_research = 30
+			engineering_research = 10
+			alloys = 3
+			minerals = 8
+			food = 1
+		}
+	}
+	
+#	potential = {
+#		has_planet_flag = exodus_happened_here
+#	}
+
+	drop_weight = {
+		weight = 0		
+	}
+}
+
+HFE_TSP_exodus_new_starting_planet = {
+	icon = "unused/d_colony_settlement"
+	use_for_min_max_adjustments = yes
+	is_for_colonizeable = yes
+	should_swap_deposit_on_terraforming = no
+	use_weights_for_terraforming_swap_types = no
+	category = deposit_cat_rare
+	triggered_planet_modifier = {
+		potential = {
+			owner = {
+				OR = {
+					has_origin = hfe_civic_features_exodus
+					has_country_flag = hfe_chosen_to_exodus
+					has_country_flag = was_government_in_exile
+				}
+			}
+			OR = {
+				has_planet_flag = hfe_features_exodus_planet
+				has_planet_flag = exodus_capital_selected
+			}
+			has_global_flag = hfe_hardcore_disable
+		}
+		district_generator_max = 5
+		district_mining_max = 5
+		district_farming_max = 5
+		planet_max_districts_add = 2
+		planet_clear_blocker_time_mult = -0.8
+	}
+
+	triggered_planet_modifier = {
+		potential = {
+			OR = {
+				owner = {
+					NOT = {
+						has_origin = hfe_civic_features_exodus
+						has_country_flag = hfe_chosen_to_exodus
+						has_country_flag = was_government_in_exile
+					}
+				}
+				NOT = { has_global_flag = hfe_hardcore_disable }
+			}
+			OR = {
+				has_planet_flag = hfe_features_exodus_planet
+				has_planet_flag = exodus_capital_selected
+			}
+		}
+		planet_housing_mult = 0.1
+		country_energy_produces_add = 10
```

Comment:
country_energy_produces_add
Shouldn't it be coutry_base?

Reply:
No. The Deposit is separate from the actual nation. Remember, if another nation captures an Exodus planet, they keep that Deposit. It is tied directly to the Deposit and nothing else. There are only two base Energy Produces Addition commands.